explorer/explorerroutes.go

```diff
@@ -239,14 +239,20 @@ func (exp *explorerUI) Mempool(w http.ResponseWriter, r *http.Request) {
 
 // Ticketpool is the page handler for the "/ticketpool" path
 func (exp *explorerUI) Ticketpool(w http.ResponseWriter, r *http.Request) {
-	chartData, groupedTickets, err := exp.explorerSource.TicketPoolVisualization(
-		dbtypes.ChartGroupingFromStr("all"),
-	)
-	if err != nil {
-		log.Errorf("Template execute failure: %v", err)
-		exp.StatusPage(w, defaultErrorCode, defaultErrorMessage, ErrorStatusType)
-		return
+	var err error
+	interval := dbtypes.ChartGroupingFromStr("all")
```

Comment:
`interval := dbtypes.AllChartGrouping` is fine in this case since the input is a constant.

Reply:
ok. Noted